Object Store/func.yaml

```diff
@@ -6,5 +6,6 @@ entrypoint: /python/bin/fdk /function/func.py handler
 memory: 1024
 timeout: 120
 config:
-  DATADOG_HOST: https://http-intake.logs.datadoghq.com/v1/input
-  DATADOG_TOKEN: <DATADOG_API_TOKEN>
+  DATADOG_HOST: https://http-intake.logs.datadoghq.com/v1/input     # DD Log Intake Host
+  DATADOG_TOKEN: <DATADOG_API_TOKEN>                                # DD API Token
+  # DATADOG_TAGS: "prod:true"                                        # Tags associated with logs
```

Comment:
I dont have context here but should this be commented out?

Reply:
This was commented out since this is meant to be user supplied tags that they can specify. I commented out here to provide an example of tags the user can specify in their logs.